### ISSUE TITLE
Out-GridView Window appears under the current active Window

### ISSUE THREAD

**Issue by Sergong** [charlieschmidt/PSmacOS#10]
First of all thanks for making this available for us MacOS users! Very useful!!
On my Mac (10.14.2), running PowerShell Core 6.1, the Out-GridView Window always appears under the current window for some reason. It would be great if it would appear as the Top Most Window. This happens with either a regular Terminal Window or the Microsoft Code Integrated Terminal.

**Comment by charlieschmidt** [comment]
Yeah aware of several window placement issues with the gridview, including remembering size & location.  On to the backlog!

THanks
